dice.js

```diff
@@ -0,0 +1,32 @@
+const rollDice = function () {
+    let num1 = Math.floor(Math.random() * 6);
```

Comment:
the die rolls should be from 1 to 6, this die roll goes from 0 to 5. Adding 1 to the variable will solve the issue